pkg/api/virtlet.k8s/v1/virtletconfig.go

```diff
@@ -68,6 +68,8 @@ type VirtletConfig struct {
 	LogLevel *int `json:"logLevel,omitempty"`
 	// Kubelet's root dir
 	KubeletRootDir *string `json:"kubeletRootDir,omitempty"`
+	// Kubelet's root dir
```

Comment:
The comment needs to be changed.

Reply:
Thanks for catching up on this. updated.